### PR TITLE
vision: Mention the frame argument

### PR DESCRIPTION
**Pull request by trickeydan**

### CODE REVIEW

File: programming/sr/cheat_sheet.md [hunk 1]
```diff
@@ -191,6 +191,26 @@ for marker in markers:
...
+<div class="info">
+The <code>frame=</code> argument is only available from software version <code>2023.1.0</code> and later.
+</div>
+
+You can use this frame in your own vision processing, or use it with other vision commands.
+
+~~~~~ python
+markers = R.camera.see(frame=frame)
+marker_ids = R.camera.see_ids(frame=frame)
+marker_ids = R.camera.save("photo.jpg", frame=frame)
```
Comment (raccube) [inline programming/sr/cheat_sheet.md:211]: The `save` method doesn't return anything. See #374

File: programming/sr/vision/index.md [hunk 2]
```diff
@@ -81,6 +81,18 @@ frame = R.camera.capture()
...
+<div class="info">
+The <code>frame=</code> argument is only available from software version <code>2023.1.0</code> and later.
+</div>
+
+You can also use a frame with other vision commands to avoid recapturing. This may be useful if you wish to use both your own vision algorithms and our vision on the same frames.
+
+~~~~~ python
+markers = R.camera.see(frame=frame)
+marker_ids = R.camera.see_ids(frame=frame)
+marker_ids = R.camera.save("photo.jpg", frame=frame)
```
Comment (raccube) [inline programming/sr/vision/index.md:93]: See other comment.

File: programming/sr/cheat_sheet.md [hunk 3]
```diff
@@ -191,6 +191,26 @@ for marker in markers:
...
+
+It is also possible to get an OpenCV frame from the camera:
+
+~~~~~ python
+frame = R.camera.capture()
+~~~~~
+
+<div class="info">
+The <code>frame=</code> argument is only available from software version <code>2023.1.0</code> and later.
+</div>
```
Comment (PeterJCLaw) [inline programming/sr/cheat_sheet.md:204]: Presumably this isn't needed now that SR2023 is complete?
```suggestion
```
(also elsewhere)

File: programming/sr/vision/index.md [hunk 4]
```diff
@@ -81,6 +81,18 @@ frame = R.camera.capture()
 flipped = cv2.flip(frame, 0)
 ~~~~~
 
+<div class="info">
+The <code>frame=</code> argument is only available from software version <code>2023.1.0</code> and later.
+</div>
+
```
Comment (PeterJCLaw) [inline programming/sr/vision/index.md:87]:
```suggestion
```
(see other comment)

**Review by PeterJCLaw — COMMENTED** [review]
Nice. Some minor fixes, but it would be great to get this shipped.

**Comment by PeterJCLaw** [comment]
Replaced by #498.

**Pull request closed without merging**